jenkins-compose.sh

```diff
@@ -17,19 +31,48 @@ cd $(dirname $(readlink -f $0 || realpath $0))
 # we don't use usual .env filename, because docker-compose uses it
 [[ -f env.local ]] && source env.local
 
+############################################################################
+# Handle deprecated/backward-compatible and default variables
+############################################################################
+
+# warning: always make sure to self-reference the variable before to allow sourcing with overrides
+
+# note: the 'SSH' and 'SLAVE' are purposely swapped below to match the legacy name
+export JENKINS_SLAVE_SSH_PUBKEY=${JENKINS_SLAVE_SSH_PUBKEY:-${JENKINS_SSH_AGENT_PUBKEY}}
+export JENKINS_SSH_SLAVE_PUBKEY=${JENKINS_SSH_SLAVE_PUBKEY:-${JENKINS_SLAVE_SSH_PUBKEY}}
```

Comment:
Wait I am lost, there are 2 flavors of `SLAVE` `PUBKEY` now?  There was just 1 before !